### PR TITLE
Add automated tests

### PR DESCRIPTION
**Pull request by Chithrak07**

### CODE REVIEW

File: molecule/default/converge.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+- name: Converge
+  hosts: all
+  become: true
+  vars:
```
Comment (dale-c-anderson) [inline molecule/default/converge.yml]: Do we actually need these 3 vars here? The role's own defaults already specify these. 

Speaking of which, 

Roll the version and the checksum in the defaults to that of 2.4.3. Version 2.4.2 failed to compile in my local docker test but 2.4.3 worked OK, and 2.4.3 was only a minor bugfix, so won't cause any breaking changes.

**Comment by dale-c-anderson** [comment]
For this role, is there a trivial way to test that zint works after it's installed in a verify.yml? I don't actually know what it does / how it works.

**Comment by Chithrak07** [comment]
> For this role, is there a trivial way to test that zint works after it's installed in a verify.yml? I don't actually know what it does / how it works.

Added a verification to check if zint --help works.

File: molecule/default/converge.yml (hunk 1 above)
Comment (Chithrak07) [reply molecule/default/converge.yml]: Done.

**Review by dale-c-anderson — APPROVED** [review]
:+1:

**Review by dale-c-anderson — APPROVED** [review]
:+1:

**Pull request merged**